src/Microsoft.ML.StaticPipe/NormalizerStaticExtensions.cs

```diff
@@ -16,61 +16,61 @@ namespace Microsoft.ML.StaticPipe
     /// </summary>
     public static class NormalizerStaticExtensions
     {
-        private const long MaxTrain = NormalizingEstimator.Defaults.MaxTrainingExamples;
+        private const long MaxTrain = NormalizingEstimator.Defaults.MaximumExampleCount;
         private const bool FZ = NormalizingEstimator.Defaults.FixZero;
 
         /// <summary>
         /// Learns an affine function based on the minimum and maximum, so that all values between the minimum and
         /// maximum observed during fitting fall into the range of -1 to 1.
         /// </summary>
         /// <param name="input">The input column.</param>
-        /// <param name="fixZero">If set to <c>false</c>, then the observed minimum and maximum during fitting
+        /// <param name="ensureZeroUntouched">If set to <c>false</c>, then the observed minimum and maximum during fitting
         /// will map to -1 and 1 respectively, exactly. If however set to <c>true</c>, then 0 will always map to 0.
         /// This is valuable for the sake of sparsity preservation, if normalizing sparse vectors.</param>
-        /// <param name="maxTrainingExamples">When gathering statistics only look at most this many examples.</param>
+        /// <param name="maximumExampleCount">When gathering statistics only look at most this many examples.</param>
         /// <param name="onFit">A delegate that can be called whenever the function is fit, with the learned slopes
-        /// and, if <paramref name="fixZero"/> is <c>false</c>, the offsets as well.</param>
+        /// and, if <paramref name="ensureZeroUntouched"/> is <c>false</c>, the offsets as well.</param>
         /// <remarks>Note that the statistics gathering and normalization is done independently per slot of the
         /// vector values.
         /// Note that if values are later transformed that are lower than the minimum, or higher than the maximum,
         /// observed during fitting, that the output values may be outside the range of -1 to 1.</remarks>
         /// <returns>The normalized column.</returns>
         public static NormVector<float> Normalize(
-            this Vector<float> input, bool fixZero = FZ, long maxTrainingExamples = MaxTrain,
+            this Vector<float> input, bool ensureZeroUntouched = FZ, long maximumExampleCount = MaxTrain,
```

Comment:
>ensureZeroUntouched [](start = 43, length = 19)

I'd prefer to be consistent throughout the APIs on what we call this.

Reply:
`bool fixZero` are all `ensureZeroUntouched`

---
In reply to: [264867559](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264867559) [](ancestors = 264867559)